packages/vm/src/evm/evm.ts

```diff
@@ -22,7 +22,7 @@ import EEI from './eei'
 import { ERROR, VmError } from '../exceptions'
 import { default as Interpreter, InterpreterOpts, RunState } from './interpreter'
 import Message, { MessageWithTo } from './message'
-import * as eof from './opcodes/eof'
+import EOF from './eof'
```

Comment:
Ah, yes, that's better. I was actually me who introduced the structure if I remember correctly.